active.cabal

```diff
@@ -20,25 +20,25 @@ source-repository head
 
 library
   exposed-modules:     Data.Active
-  build-depends:       base >= 4.0 && < 4.11,
+  build-depends:       base >= 4.0 && < 4.16,
```

Comment:
Do we need to update the lower bound too?

Reply:
I don't think so, since nothing in the code changed that would stop it working on the earlier version of `base`?

Reply:
Ah, OK.  Though I honestly have no idea if it does still work all the way back to base 4.0.